tools/tesseratos/include/tesseratos/metrics_panel/plugin.hpp

```diff
@@ -0,0 +1,24 @@
+/// @dir
+/// @brief @ref tesseratos-metrics-panel-plugin plugin directory.
+
+/// @file
+/// @brief Plugin entry point.
+/// @ingroup tesseratos-metrics-panel-plugin
+
+#pragma once
+
+#include <cubos/engine/cubos.hpp>
+
+namespace tesseratos
+{
+    /// @defgroup tesseratos-metrics-panel-plugin Metrics Panel
+    /// @ingroup tesseratos
+    /// @brief Shows Game Metrics
+    ///
+    /// This plugin shows useful performance metrics, such as fps and eleapsed time.
```

Comment:
This shouldn't be here I suppose.

Reply:
I'll delete it, but y not? the other plugin scripts also have it no?